src/locale/messages.en-US.xlf

```diff
@@ -402,6 +402,42 @@
         <source>notification-icon.toggle-panel.tooltip</source>
         <target state="new">Click to toggle notification panel</target>
       </trans-unit>
+      <trans-unit id="3299373246271290710" datatype="html">
+        <source>action-triggers.action.label</source>
+        <target state="new">Action</target>
+      </trans-unit>
+      <trans-unit id="719177318571875364" datatype="html">
+        <source>common.form-required-field.button-tooltip</source>
+        <target state="new">You have not filled in a required field</target>
+      </trans-unit>
+      <trans-unit id="5742386463554387927" datatype="html">
+        <source>action-triggers.shortcut.error</source>
+        <target state="new">Shortcut is required field</target>
+      </trans-unit>
+      <trans-unit id="6359779760013296434" datatype="html">
+        <source>action-triggers.trigger-on.error</source>
+        <target state="new">Trigger on is required field</target>
```

Comment:
I don't see why this one i necessary? At the moment you cannot have this field be empty. 
In any case it should probably say:
`Please select how the shortcut should trigger`

Reply:
Changed

Reply:
> I don't see why this one i necessary? At the moment you cannot have this field be empty.

You can't set it to empty, but it starts empty.